GooglePlayInstant/UnityPlayerHelper.cs

```diff
@@ -32,5 +32,22 @@ public static AndroidJavaObject GetCurrentActivity()
                 return unityPlayer.GetStatic<AndroidJavaObject>("currentActivity");
             }
         }
+
+        public static bool IsInstantApp () {
```

Comment:
In general I've been using `#if PLAY_INSTANT` for this purpose.

If you want to also have a method, then I'd recommend this implementation:

```
        /// <summary>
        /// Return true if this is an instant app build, false if an installed app build.
        /// </summary>
        public static bool IsInstantApp()
        {
#if PLAY_INSTANT
            return true;
#else
            return false;
#endif
        }
```